chain/chain/src/store/mod.rs

```diff
@@ -418,67 +418,15 @@ impl ChainStore {
     }
 
     pub fn reassign_outgoing_receipts_for_resharding(
-        receipts: &mut Vec<Receipt>,
+        _receipts: &mut Vec<Receipt>,
         protocol_version: ProtocolVersion,
-        shard_layout: &ShardLayout,
+        _shard_layout: &ShardLayout,
         shard_id: ShardId,
         receipts_shard_id: ShardId,
     ) -> Result<(), Error> {
         tracing::trace!(target: "resharding", ?protocol_version, ?shard_id, ?receipts_shard_id, "reassign_outgoing_receipts_for_resharding");
-        // If simple nightshade v2 is enabled and stable use that.
-        // Same reassignment of outgoing receipts works for simple nightshade v3
-        if ProtocolFeature::SimpleNightshadeV2.enabled(protocol_version) {
-            Self::reassign_outgoing_receipts_for_resharding_v2(
```

Comment:
Undo changes here and retain this block of code. Rename `reassign_outgoing_receipts_for_resharding_v2` to `reassign_outgoing_receipts_for_resharding_impl`